test/distributed_ranges/include/common_tests.hpp

```diff
@@ -217,7 +213,7 @@ std::string check_segments_message(auto &&r) {
                        "      segments: {}\n  ",
                        stdrng::views::all(r), stdrng::views::all(segments));
   }
-  return "";
+  return drfmt::format("");
```

Comment:
without fmt all checks in tests fail

it is because lines like this returning non-empty (check failed) even if you pass "" to it